lesson3.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('js-list');
+for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    const img = document.createElement('img');
+    const imgbox = ['/img/bookmark.png','/img/message.png'];
+    img.src = imgbox[i-1];
```

Comment:
[IMO].
I think using arrays is a good idea.

Let's figure out how to make it better.
The /img/ defined by an array of constant img can be common.
So it looks like we can iterate with template literals in img.src used in the loop statement.

Reply:
This way of thinking. I learned a lot from it. I will continue to work on making it simple to describe.